contributing/community/review-comments.rst

```diff
@@ -113,7 +113,7 @@ If a piece of code is in fact wrong, explain why:
 
     * I profiled this change and it hurts performance significantly (if you don't profile, it's an opinion, so we can ignore)
 
-    * Code doesn't match Symfony's CS rules (e.g. ``use array()`` instead of ``[]``)
```

Comment:
We must use another example here now that the short syntax is standard.

Reply:
Can't we just swap it?
```suggestion
    * Code doesn't match Symfony's CS rules (e.g. use ``[]`` instead of ``array()``)
```

Reply:
@wouterj that's why i love you so much.